lib/Projects/GetProjectPackagistData.php

```diff
@@ -12,15 +12,19 @@ class GetProjectPackagistData
 {
     private const PACKAGIST_URL_FORMAT = 'https://packagist.org/packages/%s.json';
```

Comment:
This constant is no longer in use I think.

Reply:
It's used in the constructor as a default value. I made it injectable to be able to test the class. I can remove the default value in the constructor and put the url into the service definitions.